src/main/java/com/example/gradecalculator/model/UserDetailsImpl.java

```diff
@@ -52,4 +52,6 @@ public boolean isEnabled() {
         return true;
     }
 
+    public Long getId() { return user.getId(); }
```

Comment:
everything else is multi-line - to keep it readable your code needs to use space ;)